projects/uswds-formly/src/lib/types/range-slider.ts

```diff
@@ -0,0 +1,21 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { FieldType } from '@ngx-formly/core';
+
+@Component({
+  template: `
+  <usa-range-slider
+    [formControl]="formControl"
+    [min]="to.min ? to.min :  0"
+    [max]="to.max ? to.max : 100"
+    [step]="to.step ? to.step : 10"
+    [startingValue]="to.startingValue ? to.startingValue : 0"
+    [formlyAttributes]="field"
+    [id]="to.id ? to.id : 'usa-range'"
+
+  ></usa-range-slider>
```

Comment:
If we do remove the label from the component, could we add a label here if there isn't a wrapper for it?